### PR TITLE
Add Initial support for pre-commit hooks

### PR DESCRIPTION
**Pull request by VMRuiz**
Fixes https://github.com/scrapy/itemloaders/issues/69

### CODE REVIEW

**Comment by wRAR** [comment]
Can you please commit the `black` changes separately and add that commit to `.git-blame-ignore-revs`?

File: requirements-dev.txt [hunk 1]
```diff
@@ -3,5 +3,8 @@ parsel>=1.5.2
 jmespath>=0.9.5
 itemadapter>=0.1.0
 
-pytest==5.4.1
```
Comment (wRAR) [inline requirements-dev.txt:6]: I wonder what is this file about tbh. It's not mentioned anywhere and I don't think it's needed for development.

Comment (VMRuiz) [reply requirements-dev.txt:6]: The "requirements-dev.txt" file contains all the necessary packages for local development, which you need to install in your virtual environment to work with the package. 

Although Pytest may already be installed in the global system,but I prefer to have it in the local env too. Also, while pre-commit installs Black, isort, and flake8 automatically, it's helpful to have them installed, in order to run them independently. That said, if you prefer to remove them, you can do so.

**Review by Gallaecio — APPROVED** [review]

File: requirements-dev.txt (hunk 1 above)
Comment (kmike) [reply requirements-dev.txt:6]: Shouldn't you use tox, not pytest locally?

Comment (kmike) [reply requirements-dev.txt:6]: I'm actually not sure why we need requirements-dev.txt :)

Comment (Gallaecio) [reply requirements-dev.txt:6]: PS: https://github.com/scrapy/itemloaders/blob/master/tox.ini seems outdated on the Python version front.

**Review by serhii73 — APPROVED** [review]

**Pull request merged**